src/index.html

```diff
@@ -1,16 +1,23 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>Mate academy</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="form" 
+    data-qa="big">
+      <input type="text" class="input" 
+    placeholder= "Try &quot;Los Angeles&quot;" data-qa="keypress">
+    </form>
+
+    <form action="#" 
+    class="forms" data-qa="small">
+      <input type="text" class="inputs" 
```

Comment:
create 1 common class for inputs & forms, and create separate extra classes like `input-big or inputs-small`, to prevent code duplicating

Reply:
done